README.md

```diff
@@ -76,6 +76,17 @@ Decide on a function to match. These can be found in the subdirectories of `asm/
 
 Take the relevant `.s` file and pass it to [mips_to_c](https://github.com/matt-kempster/mips_to_c) ([web version](https://simonsoftware.se/other/mips_to_c.py)).
 
+You can also use mips_to_c locally by registering `mips_to_c.py` as a global command in your bash PATH:
```

Comment:
I think we should hold off on changing this part of the readme for now, becuase I have some ideas on how to handle tools across multiple projects that I'd like to try.. I'll probably make a tool that generates a context file and feeds it to mips_to_c. we can talk about it more in discord

Reply:
Anything I can do to help with that tool? What sort of context file? Cool, @ me on discord and I'll check in later. I've been able to I make time for decomp over the weekends. I can respond to messages most nights.